__tests__/index.js

```diff
@@ -0,0 +1,113 @@
+/* global describe, expect, it */
+
+import fs from 'fs'
+import path from 'path'
+
+import accessibilityForGrid from '../lib/accessibility-for-grid'
+import generateDestinationData from '../lib/generate-destination-data'
+import getIsochrone from '../lib/get-isochrone'
+import getSurface from '../lib/get-surface'
+import {create as createGrid} from '../lib/grid'
+import {create as createOrigin} from '../lib/origin'
+import {create as createStopTreeCache} from '../lib/stop-tree-cache'
+
+const DATA = path.join(__dirname, '/data')
+const DEST_POINT = {x: 158, y: 140}
+const ORIGIN_POINT = {x: 125, y: 136}
+const which = 'MEDIAN'
+
+const jobsGrid = fs.readFileSync(`${DATA}/jobs.grid`).buffer
+const originData = fs.readFileSync(`${DATA}/origin.dat`).buffer
+const query = JSON.parse(fs.readFileSync(`${DATA}/query.json`, 'utf8'))
+const stopTrees = fs.readFileSync(`${DATA}/stop_trees.dat`).buffer
+
+const ctx = {}
+
+describe('Browsochrones', () => {
+  it('create stop tree cache', () => {
+    ctx.stopTreeCache = createStopTreeCache({
+      data: new Int32Array(stopTrees),
+      size: query.width * query.height
+    })
+    const {index} = ctx.stopTreeCache
```

Comment:
Should we check certain percentiles? A set of random numbers decided up front? I added simple-statistics below as I had the thought just to snapshot summary stats of the data but then decided against it.